jqqb/rule.py

```diff
@@ -26,38 +28,46 @@ def values(self):
         values = self.get_value()
         return values
 
+    def values_custom_fields(self, obj):
+        values = self.get_value_custom_fields(obj)
+        return values
+
     def inspect(self, obj):
-        return self.inputs(obj), self.values(), self.evaluate(obj)
+        return self.inputs(obj), self.values_custom_fields(obj), self.evaluate(obj)
 
     def get_operator(self):
         return getattr(Operators, "eval_" + self.operator)
 
-    def get_input(self, obj):
-        fields = self.field.split(".")
+    def get_result(self, obj, input):
```

Comment:
`field` seems better to name this variable.

Reply:
`get_result` make us understand you resolve the phrase, but it is not.
What do you think about `get_field_value`?